test/e2e/lib/flows/gutenboarding-flow.js

```diff
@@ -30,15 +30,15 @@ export default class CreateSiteFlow {
 			await acquireIntentPage.skipStep();
 		}
 
+		const domainsPage = await DomainsPage.Expect( this.driver );
+		await domainsPage.skipStep();
+
 		const designSelectorPage = await DesignSelectorPage.Expect( this.driver );
 		await designSelectorPage.selectFreeDesign();
 
 		const stylePreviewPage = await StylePreviewPage.Expect( this.driver );
 		await stylePreviewPage.continue();
 
-		const domainsPage = await DomainsPage.Expect( this.driver );
-		await domainsPage.skipStep();
-
```

Comment:
This change will break `wp-launch-spec` and `wp-signup-spec` where `name` is `undefined` so we're skipping the `acquireIntentPage` (see line 30 above). 
In that case, `DomainsPage` is not the second step but the one displayed after selecting design and font. See https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/gutenboarding/hooks/use-steps.ts#L27-L45

Reply:
Yup thanks for the pointers. It'll now generate a fallback when no site name is given.